sdk/core/core-tracing/package.json

```diff
@@ -32,7 +32,7 @@
     "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "echo skipped",
-    "unit-test:node": "mocha test-dist/**/*.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
+    "unit-test:node": "mocha test-dist/**/*.js --reporter ../../../common/tools/mocha-multi-reporter.js",
```

Comment:
- The `--reporter` options are exactly same for all the SDKs.
- For local runs - we only need spec reporter  and not junit-reporter.
- `spec reporter` is the default reporter when we don't specify a reporter.
- Both junit-reporter and spec reporter are required in the CI. 

One proposal is to move the reporter options to the rush-yaml scripts instead of `package.json`s if possible.
@KarishmaGhiya is checking if rush and our configs allow this.